docs/Makefile

```diff
@@ -18,7 +18,7 @@ clean:
 
 .PHONY: html
 html:
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html $(SPHINXOPTS)
+	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html $(SPHINXOPTS)  -j 'auto'
```

Comment:
Its cool that this sped up the `build_docs` job :) out of curiosity do you know by how much?

Reply:
It goes from about 8-9 minutes before this change to ~6 minutes after